### PR TITLE
Create a constant for waiting times

### PR DESCRIPTION
**Pull request by pritalpatel**
Pull Request for Issue #45 .

### CODE REVIEW

File: tests/acceptance/_bootstrap.php [hunk 1]
```diff
@@ -1 +1,3 @@
 <?php
+define("TIMEOUT", 60);
```
Comment (yvesh) [inline tests/acceptance/_bootstrap.php:2]: Add a check if it is not already set (common name)

``` php
if (!defined('TIMEOUT')) 
{
```

File: tests/_support/Step/Acceptance/Administrator/Content.php [hunk 2]
```diff
@@ -42,7 +42,7 @@ public function iSaveAnArticle()
 	public function iShouldSeeTheMessage($message)
 	{
 		$I = $this;
-		$I->waitForText($message, 60, AdminPage::$systemMessageContainer);
```
Comment (javigomez) [inline tests/_support/Step/Acceptance/Administrator/Content.php:45]: I think we would need to modify all the Wait commands in the whole test suite. For example: https://github.com/joomla-projects/gsoc16_browser-automated-tests/blob/staging/tests/acceptance/administrator/UserCest.php#L36

so any waitForText or waitForElement in all the step files should use the TIMEOUT constant

File: tests/acceptance/_bootstrap.php (hunk 1 above)
Comment (javigomez) [inline tests/acceptance/_bootstrap.php:2]: I would also add a /tests/readme.md file explaining the particular details of the tests. In this case you can say:
- you can modify the timeout time by changing the value of this constant... lower for fast machines and higher for slow computers...

File: tests/_support/Step/Acceptance/Administrator/Content.php (hunk 2 above)
Comment (pritalpatel) [inline tests/_support/Step/Acceptance/Administrator/Content.php:45]: I have tried to find all the other files for timout, I think this is the only place I found so far. I will add and use `TIMEOUT` constant to replace all the static time. Thank you.

File: tests/acceptance/_bootstrap.php (hunk 1 above)
Comment (pritalpatel) [inline tests/acceptance/_bootstrap.php:2]: Sure. I will add the same in README.md and I will also add condition like suggested by Yves.

**Comment by javigomez** [comment]
I'm merging this one and creating an issue so you can continue your work in a new pull.

**Pull request merged**